app/widgets/common/deploymentsView/header/RunWorkflowModal.tsx

```diff
@@ -125,12 +130,34 @@ const RunWorkflowModal: FunctionComponent<RunWorkflowModalProps> = ({ filterRule
         turnOffLoading();
     }
 
+    const initializeParametersInputs = () => {
+        const defaultParametersData = selectedWorkflow?.parameters
+            ? selectedWorkflow.parameters.reduce((parameters, parameter) => {
+                  parameters[parameter.name] = parameter.default;
+                  return parameters;
+              }, {} as Record<string, unknown>)
```

Comment:
Optional:

Depending on how you address comments in `useParametersInputs` maybe worth to reuse type from there:
```suggestion
              }, {} as ParameterInputs)
```
? 🤔

Reply:
Sure 💪 , extracted the type and reused it within b36a865bf3ecbe7dcca77b7ff5579e3fdb5121a4